dot/types/digest.go

```diff
@@ -15,11 +16,40 @@ func NewDigestItem() scale.VaryingDataType {
 	return scale.MustNewVaryingDataType(ChangesTrieRootDigest{}, PreRuntimeDigest{}, ConsensusDigest{}, SealDigest{})
 }
 
-// NewDigest returns a new Digest from the given DigestItems
+// NewDigest returns a new Digest as a varying data type slice.
 func NewDigest() scale.VaryingDataTypeSlice {
 	return scale.NewVaryingDataTypeSlice(NewDigestItem())
 }
 
+// DigestToString renders a digest varying data type slice as a string.
+func DigestToString(digest scale.VaryingDataTypeSlice) (s string) {
```

Comment:
is it possible to create a `String` method  at `VaryingDataTypeSlice` struct?

Reply:
Yes, done in https://github.com/ChainSafe/gossamer/pull/2970 :+1: